pages/projects/[id]/index.js

```diff
@@ -17,10 +17,19 @@ export default function ProjectDetailPage() {
 
   const { title, description, endDate, department, teamLead } = project;
 
+  async function handleDelete() {
+    await fetch(`/api/projects/${id}`, {
+      method: "DELETE",
+    });
+
+    router.push("/");
+  }
+
   return (
     <>
       <Button />
```

Comment:
My suggestion would be, to rename your `Button` to `BackLink` or something, without looking inside your Button component, you don't know, for what that button is.